pkg/preflight/assets/host-preflights.yaml

```
@@ -79,8 +79,26 @@ spec:
         backgroundWriteIOPSJobs: 6
         backgroundReadIOPS: 50
         backgroundReadIOPSJobs: 1
-
+    - http:
+        collectorName: "Kubernetes API healthz endpoint"
+        exclude: '{{kurl and .IsPrimary | not }}'
```

Will this work on new installs? Don't preflights run before we install kubernetes? Perhaps you mean to add .isUpgrade here?

```suggestion
        exclude: '{{kurl or (not .IsPrimary) (not .IsUpgrade) }}'
```

---

No, It will not. Thank you for help out on this.
I got how that works now either.